models/attendees.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
Please rename this file from models/attendees.js to models/attendee.js so that the filename is singular as well.